webapp/src/pages/eventHome/EventHome.js

```diff
@@ -154,6 +154,11 @@ class EventHome extends Component {
           path={`${match.path}/apply`}
           render={(props) => <Application {...props} event={event} />}
         />
+        <Route
+          exact
+          path={`${match.path}/apply/newSubmission`}
```

Comment:
Mind making this a bit more generic, like call it apply/new  (rather than newSubmission ?) We will probably want to do the same for nominations, allowing a direct link to add a new one instead of going to the list :) Thanks!

Reply:
Done! :)